src/main/java/io/appium/java_client/ios/PerformsTouchID.java

```diff
@@ -35,9 +35,24 @@ default void performTouchID(boolean match) {
 
     /**
      * Enrolls touchId in iOS Simulators.
-     *
+     * This method is deprecated. Please use {@link #toggleTouchIDEnrollment(boolean)} instead
      */
+    @Deprecated
     default void toggleTouchIDEnrollment() {
```

Comment:
@mykola-mokhnach We haven't released a version after this implementation. Can we remove this or Do you still wanted it to be marked `deprecated`?